### PR TITLE
Updated the Sorting Formula

### PR DESCRIPTION
**Pull request by lgarceau768**
Updated the formula and removed unused variables
Updated the formula for sorting to `1 - (matchingCriteria / totalCriteria) + (ifIneligble ? 2048: 0)`
This is a fix to #510

### CODE REVIEW

**Review by wesley-dean-gsa — APPROVED** [review]
Approved.

However, just wondering if it's possible for `benefit.eligibility.length - virtualBenefitEligibility` to ever be zero (leading to a divide by zero condition).

**Review by fpigeonjr — APPROVED** [review]
worked well in your demo this morning.

File: pages/_lifeEvent.vue [hunk 1]
```diff
@@ -340,15 +340,11 @@ export default {
         this.lifeEventBenefits = _.sortBy(this.lifeEventBenefits, [this.sort])
       } else {
         const forceToBottom = 2048
-        const virtualCriteria = this.getVirtualCriteria()
         this.lifeEventBenefits.forEach((benefit) => {
           const matches = (benefit.matches = this.getTotalEligibleCriteria(benefit.eligibility))
-          const ineligible = this.getTotalIneligibleCriteria(benefit.eligibility) > 0
-          const virtualBenefitEligibility = (benefit.virtualBenefitEligibility = (benefit.eligibility || []).filter(
```
Comment (wesley-dean-gsa) [inline pages/_lifeEvent.vue:347]: Tangentially-related to the PR, is there a way to simplify this and make it more readable?  It hurts my brain.

Comment (lgarceau768) [reply pages/_lifeEvent.vue:347]: We could split up the `+ (ineligible ? forceToBottom: 0)` into another separate variable if that would help 👍🏼

**Pull request merged**